hs_core/management/commands/solr_prepare.py

```diff
@@ -24,8 +24,8 @@ def debug_harvest():
         pprint(ind.prepare_title(obj))
```

Comment:
From Alva: this needs to be reverted to the prior version. I reverted most files but this one I forgot.